src/main.ts

```diff
@@ -10,6 +10,7 @@ import * as cookieParser from 'cookie-parser';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule, { bufferLogs: true });
+  app.enableCors();
```

Comment:
What is your view on setting an array of valid origins? For example, localhost:5000 in development env or universiteams.com in production env
By using this configuration object: https://github.com/expressjs/cors#configuration-options as enableCors() argument.

Reply:
The default setting makes all origins accepted by Cors (origin: "*") and that is a potential security vulnerability. You are right, I better add an array of possible origins